system/Database/SQLSRV/Builder.php

```diff
@@ -199,6 +201,26 @@ protected function _insert(string $table, array $keys, array $unescapedKeys): st
         return $this->keyPermission ? $this->addIdentity($fullTableName, $statement) : $statement;
     }
 
+    //--------------------------------------------------------------------
+
+    /**
+     * Insert batch statement
+     *
+     * Generates a platform-specific insert string from the supplied data.
+     *
+     * @param string $table  Table name
+     * @param array  $keys   INSERT keys
+     * @param array  $values INSERT values
+     *
+     * @return string
+     */
+    protected function _insertBatch(string $table, array $keys, array $values): string
+    {
+        return 'INSERT ' . $this->compileIgnore('insert') . 'INTO ' . $this->db->escapeIdentifiers($this->db->schema) . '.' . $table . ' (' . implode(', ', $keys) . ') VALUES ' . implode(', ', $values);
```

Comment:
Any reason why we're using `$this->db->schema` instead of `getFullName()` method? I mean not only here but in general - in the `Builder` class.

Reply:
In the Builder class we can probably switch back to using `getFullName()` in the other classes there are different ways how the schema is used in raw sql. Sometimes as string sometimes as object identifier.
Once i was in the flow with the other classes i just kept my changes aligned to use `$this->db->schema` or `$db->schema`.

Reply:
I get it :)
Since we already have this method, I think it would be a good idea to use it - only in `Builder` class of course.

Reply:
I will clean this up and add it back, will probably be in the tomorrow